test/features/deploy_springboot_service.feature

```diff
@@ -59,3 +59,29 @@ Feature: Deploy spring boot service
     # Implement retrieving of job information from Jobs service once https://issues.redhat.com/browse/KOGITO-1163 is fixed
     Then Kogito application "process-timer-springboot" log contains text "Before timer" within 1 minutes
     And Kogito application "process-timer-springboot" log contains text "After timer" within 1 minutes
+
+#####
+
+  @events
+  @persistence
+  Scenario: Process instance events are stored in Data Index
```

Comment:
Add `Spring Boot` into the title, so we can differentiate them

Reply:
Done